mh/constants.py

```diff
@@ -7,21 +7,26 @@
 import struct
 from other.utils import pad
 from mh.time_utils import current_server_time, TICKS_PER_CYCLE, \
-    get_jhen_event_times, is_jhen_active
-from mh.quest_utils import \
-    QUEST_EVENT_JUMP_FOUR_JAGGI, QUEST_EVENT_BLOOD_SPORT, \
-    QUEST_EVENT_MERCY_MISSION, QUEST_EVENT_THE_PHANTOM_URAGAAN, \
-    QUEST_EVENT_WORLD_EATER, QUEST_EVENT_WHERE_GODS_FEAR_TO_TREAD, \
-    make_binary_event_quest
-from mh.arena_utils import \
-    GRUDGE_MATCH_ROYAL_LUDROTH, GRUDGE_MATCH_BIRD_BRUTE, \
-    GRUDGE_MATCH_TWO_FLAMES, GRUDGE_MATCH_LAND_LORDS
-
-
-def make_binary_type_time_events():
+    get_jhen_event_times, is_jhen_active, current_event_time_slot
+from mh.quest_utils import make_binary_event_quest, generate_flags
+
+
+def make_binary_type_time_events(binary_loader=None):
```

Comment:
Is adding a useless optional parameter necessary?

Reply:
Currently yes, since all the dynamically updating (in other words, callable instead of static) binaries have the binary loader passed in just in case it's needed (line 1140 of `pat.py`)